diff --git a/templates/vinyl/browse.html.twig b/templates/vinyl/browse.html.twig
index 4adb777..16a03b9 100644
--- a/templates/vinyl/browse.html.twig
+++ b/templates/vinyl/browse.html.twig
@@ -25,7 +25,7 @@
 
     <div>
         <h2 class="mt-5">Mixes</h2>
-        <turbo-frame id="mix-list-page-{{ pager.currentPage }}">
+        <turbo-frame id="mix-list-page-{{ pager.currentPage }}" target="_top">
             <div class="row">
                 {% for mix in pager %}
                 <div class="col col-md-4">
