diff --git a/templates/mix/addSong.html.twig b/templates/mix/addSong.html.twig
index 62e1bee..bc878a2 100644
--- a/templates/mix/addSong.html.twig
+++ b/templates/mix/addSong.html.twig
@@ -6,14 +6,16 @@
 <div class="container">
     <h1>Add a Song</h1>
 
-    {{ form_start(form) }}
-        {{ form_row(form.songTitle) }}
-        {{ form_row(form.artistName) }}
-        {{ form_row(form.length) }}
+    <turbo-frame id="add-song-form">
+        {{ form_start(form) }}
+            {{ form_row(form.songTitle) }}
+            {{ form_row(form.artistName) }}
+            {{ form_row(form.length) }}
 
-        <button type="submit" class="btn btn-primary" formnovalidate>Add <span class="fa fa-plus"></span></button>
-        <a href="{{ path('app_mix_edit', { slug: mix.slug }) }}">cancel</a>
-    {{ form_end(form) }}
+            <button type="submit" class="btn btn-primary" formnovalidate>Add <span class="fa fa-plus"></span></button>
+            <a href="{{ path('app_mix_edit', { slug: mix.slug }) }}">cancel</a>
+        {{ form_end(form) }}
+    </turbo-frame>
 </div>
 {% endblock %}
 
diff --git a/templates/mix/edit.html.twig b/templates/mix/edit.html.twig
index 3beb223..4ff1769 100644
--- a/templates/mix/edit.html.twig
+++ b/templates/mix/edit.html.twig
@@ -16,9 +16,12 @@
                     withControls: true
                 }) }}
             {% endfor %}
-            <a href="{{ path('app_mix_edit_add_song', {
-                slug: mix.slug,
-            }) }}" class="btn btn-success"><i class="fas fa-plus"></i> Add a song</a>
+
+            <turbo-frame id="add-song-form">
+                <a href="{{ path('app_mix_edit_add_song', {
+                    slug: mix.slug,
+                }) }}" class="btn btn-success"><i class="fas fa-plus"></i> Add a song</a>
+            </turbo-frame>
         </div>
     </div>
 </div>
