diff --git a/templates/mix/edit.html.twig b/templates/mix/edit.html.twig
index 4ff1769..cca6724 100644
--- a/templates/mix/edit.html.twig
+++ b/templates/mix/edit.html.twig
@@ -17,7 +17,7 @@
                 }) }}
             {% endfor %}
 
-            <turbo-frame id="add-song-form">
+            <turbo-frame id="add-song-form" data-turbo-action="advance">
                 <a href="{{ path('app_mix_edit_add_song', {
                     slug: mix.slug,
                 }) }}" class="btn btn-success"><i class="fas fa-plus"></i> Add a song</a>
