diff --git a/src/Controller/MixController.php b/src/Controller/MixController.php
index 5cdb143..03eb153 100644
--- a/src/Controller/MixController.php
+++ b/src/Controller/MixController.php
@@ -34,7 +34,9 @@ class MixController extends AbstractController
     {
         $track = new Track();
         $mix->addTrack($track);
-        $form = $this->createForm(TrackFormType::class, $track);
+        $form = $this->createForm(TrackFormType::class, $track, [
+            'action' => $request->getUri(),
+        ]);
 
         $form->handleRequest($request);
         if ($form->isSubmitted() && $form->isValid()) {
