diff --git a/src/Twig/Components/RegistrationFormComponent.php b/src/Twig/Components/RegistrationFormComponent.php
index 2ddbed1..8455211 100644
--- a/src/Twig/Components/RegistrationFormComponent.php
+++ b/src/Twig/Components/RegistrationFormComponent.php
@@ -2,11 +2,18 @@
 
 namespace App\Twig\Components;
 
+use Symfony\Component\Form\FormInterface;
 use Symfony\UX\LiveComponent\Attribute\AsLiveComponent;
+use Symfony\UX\LiveComponent\ComponentWithFormTrait;
 use Symfony\UX\LiveComponent\DefaultActionTrait;
 
 #[AsLiveComponent('registration_form')]
 final class RegistrationFormComponent
 {
     use DefaultActionTrait;
+    use ComponentWithFormTrait;
+
+    protected function instantiateForm(): FormInterface
+    {
+    }
 }
diff --git a/templates/components/registration_form.html.twig b/templates/components/registration_form.html.twig
index 75ebedd..f3cda92 100644
--- a/templates/components/registration_form.html.twig
+++ b/templates/components/registration_form.html.twig
@@ -1,3 +1,12 @@
 <div{{ attributes }}>
-    <!-- component html -->
+    {{ form_start(form) }}
+        {{ form_row(form.email) }}
+        {{ form_row(form.plainPassword, {
+            label: 'Password'
+        }) }}
+        {{ form_row(form.favoriteTrack) }}
+        {{ form_row(form.agreeTerms) }}
+
+        <button type="submit" class="btn btn-primary" formnovalidate>Register</button>
+    {{ form_end(form) }}
 </div>
diff --git a/templates/registration/register.html.twig b/templates/registration/register.html.twig
index 21b1329..4758d66 100644
--- a/templates/registration/register.html.twig
+++ b/templates/registration/register.html.twig
@@ -8,16 +8,7 @@
             <div class="col-6">
                 <h1>Register</h1>
 
-                {{ form_start(form) }}
-                    {{ form_row(form.email) }}
-                    {{ form_row(form.plainPassword, {
-                        label: 'Password'
-                    }) }}
-                    {{ form_row(form.favoriteTrack) }}
-                    {{ form_row(form.agreeTerms) }}
-
-                    <button type="submit" class="btn btn-primary" formnovalidate>Register</button>
-                {{ form_end(form) }}
+                {{ component('registration_form') }}
             </div>
         </div>
     </div>
