diff --git a/src/Twig/Components/RegistrationFormComponent.php b/src/Twig/Components/RegistrationFormComponent.php
new file mode 100644
index 0000000..2ddbed1
--- /dev/null
+++ b/src/Twig/Components/RegistrationFormComponent.php
@@ -0,0 +1,12 @@
+<?php
+
+namespace App\Twig\Components;
+
+use Symfony\UX\LiveComponent\Attribute\AsLiveComponent;
+use Symfony\UX\LiveComponent\DefaultActionTrait;
+
+#[AsLiveComponent('registration_form')]
+final class RegistrationFormComponent
+{
+    use DefaultActionTrait;
+}
diff --git a/templates/components/registration_form.html.twig b/templates/components/registration_form.html.twig
new file mode 100644
index 0000000..75ebedd
--- /dev/null
+++ b/templates/components/registration_form.html.twig
@@ -0,0 +1,3 @@
+<div{{ attributes }}>
+    <!-- component html -->
+</div>
