diff --git a/assets/controllers/vote-controller.js b/assets/controllers/vote-controller.js
index 3747359..e3bed0e 100644
--- a/assets/controllers/vote-controller.js
+++ b/assets/controllers/vote-controller.js
@@ -1,7 +1,7 @@
 import { Controller } from '@hotwired/stimulus';
 
 export default class extends Controller {
-    connect() {
-        console.log('it works!')
+    upVote() {
+        console.log('upvote!');
     }
 }
diff --git a/templates/mix/show.html.twig b/templates/mix/show.html.twig
index 9ac4a45..c7eabbb 100644
--- a/templates/mix/show.html.twig
+++ b/templates/mix/show.html.twig
@@ -17,6 +17,7 @@
                     <span>{{ mix.votes }}</span> votes
                     <button
                         class="btn btn-outline-primary"
+                        {{ stimulus_action('vote', 'upVote') }}
                     ><span class="fa fa-thumbs-up"></span></button>
                     <button
                         class="btn btn-outline-primary"
