diff --git a/assets/controllers/vote-controller.js b/assets/controllers/vote-controller.js
index 6fb4676..c722978 100644
--- a/assets/controllers/vote-controller.js
+++ b/assets/controllers/vote-controller.js
@@ -8,4 +8,10 @@ export default class extends Controller {
 
         this.totalTarget.innerHTML = total + 1;
     }
+
+    downVote() {
+        const total = parseInt(this.totalTarget.innerHTML.trim());
+
+        this.totalTarget.innerHTML = total - 1;
+    }
 }
diff --git a/templates/mix/show.html.twig b/templates/mix/show.html.twig
index b27cc74..c9ae665 100644
--- a/templates/mix/show.html.twig
+++ b/templates/mix/show.html.twig
@@ -21,6 +21,7 @@
                     ><span class="fa fa-thumbs-up"></span></button>
                     <button
                         class="btn btn-outline-primary"
+                        {{ stimulus_action('vote', 'downVote') }}
                     ><span class="fa fa-thumbs-down"></span></button>
                 </div>
 
