diff --git a/assets/controllers/vote-controller.js b/assets/controllers/vote-controller.js
index c722978..c28073c 100644
--- a/assets/controllers/vote-controller.js
+++ b/assets/controllers/vote-controller.js
@@ -4,14 +4,17 @@ export default class extends Controller {
     static targets = ['total'];
 
     upVote() {
-        const total = parseInt(this.totalTarget.innerHTML.trim());
-
-        this.totalTarget.innerHTML = total + 1;
+        this.#changeVote('up');
     }
 
     downVote() {
+        this.#changeVote('down');
+    }
+
+    #changeVote(direction) {
         const total = parseInt(this.totalTarget.innerHTML.trim());
+        const change = direction === 'up' ? 1 : -1;
 
-        this.totalTarget.innerHTML = total - 1;
+        this.totalTarget.innerHTML = total + change;
     }
 }
