diff --git a/assets/controllers/vote-controller.js b/assets/controllers/vote-controller.js
index e3bed0e..6fb4676 100644
--- a/assets/controllers/vote-controller.js
+++ b/assets/controllers/vote-controller.js
@@ -1,7 +1,11 @@
 import { Controller } from '@hotwired/stimulus';
 
 export default class extends Controller {
+    static targets = ['total'];
+
     upVote() {
-        console.log('upvote!');
+        const total = parseInt(this.totalTarget.innerHTML.trim());
+
+        this.totalTarget.innerHTML = total + 1;
     }
 }
diff --git a/templates/mix/show.html.twig b/templates/mix/show.html.twig
index c7eabbb..b27cc74 100644
--- a/templates/mix/show.html.twig
+++ b/templates/mix/show.html.twig
@@ -14,7 +14,7 @@
                 <p>{{ mix.description }}</p>
 
                 <div {{ stimulus_controller('vote') }}>
-                    <span>{{ mix.votes }}</span> votes
+                    <span {{ stimulus_target('vote', 'total') }}>{{ mix.votes }}</span> votes
                     <button
                         class="btn btn-outline-primary"
                         {{ stimulus_action('vote', 'upVote') }}
