diff --git a/assets/controllers/vote-controller.js b/assets/controllers/vote-controller.js
new file mode 100644
index 0000000..3747359
--- /dev/null
+++ b/assets/controllers/vote-controller.js
@@ -0,0 +1,7 @@
+import { Controller } from '@hotwired/stimulus';
+
+export default class extends Controller {
+    connect() {
+        console.log('it works!')
+    }
+}
diff --git a/templates/mix/show.html.twig b/templates/mix/show.html.twig
index fa8a889..9ac4a45 100644
--- a/templates/mix/show.html.twig
+++ b/templates/mix/show.html.twig
@@ -13,7 +13,7 @@
                 <h2 class="mb-4">{{ mix.trackCount }} songs <small>(genre: {{ mix.genre }})</small></h2>
                 <p>{{ mix.description }}</p>
 
-                <div>
+                <div {{ stimulus_controller('vote') }}>
                     <span>{{ mix.votes }}</span> votes
                     <button
                         class="btn btn-outline-primary"
