diff --git a/assets/test/controllers/vote-controller.test.js b/assets/test/controllers/vote-controller.test.js
new file mode 100644
index 0000000..3fbcb2a
--- /dev/null
+++ b/assets/test/controllers/vote-controller.test.js
@@ -0,0 +1,35 @@
+import { Application } from '@hotwired/stimulus';
+import { clearDOM, mountDOM } from '@symfony/stimulus-testing';
+import VoteController from '../../controllers/vote-controller';
+
+const startStimulus = () => {
+    const application = Application.start();
+    application.register('vote', VoteController);
+};
+
+describe('VoteController', () => {
+    let container;
+
+    afterEach(() => {
+        clearDOM();
+    });
+
+    it('connects', async () => {
+        container = mountDOM(`<div data-controller="vote">
+            <span data-vote-target="total">10</span> votes
+
+            <button
+                data-action="vote#upVote"
+            >Up</button>
+            <button
+                data-action="vote#downVote"
+            >Down</button>
+        </div>`);
+
+        startStimulus();
+
+        // Write a test here ...
+    });
+
+    // You can create other tests here
+});
