diff --git a/src/Form/RegistrationFormType.php b/src/Form/RegistrationFormType.php
index eca091d..68ae9ec 100644
--- a/src/Form/RegistrationFormType.php
+++ b/src/Form/RegistrationFormType.php
@@ -20,14 +20,7 @@ class RegistrationFormType extends AbstractType
     {
         $builder
             ->add('email')
-            ->add('favoriteTrack', EntityType::class, [
-                'class' => Track::class,
-                'placeholder' => 'Choose a song',
-                'choice_label' => function(Track $track) {
-                    return sprintf('%s (by %s)', $track->getSongTitle(), $track->getArtistName());
-                },
-                'autocomplete' => true,
-            ])
+            ->add('favoriteTrack', TrackAutocompleteField::class)
             ->add('agreeTerms', CheckboxType::class, [
                 'mapped' => false,
                 'constraints' => [
diff --git a/src/Form/TrackAutocompleteField.php b/src/Form/TrackAutocompleteField.php
index e9e64fe..31546a9 100644
--- a/src/Form/TrackAutocompleteField.php
+++ b/src/Form/TrackAutocompleteField.php
@@ -16,8 +16,10 @@ class TrackAutocompleteField extends AbstractType
     {
         $resolver->setDefaults([
             'class' => Track::class,
-            'placeholder' => 'Choose a Track',
-            //'choice_label' => 'name',
+            'placeholder' => 'Choose a song',
+            'choice_label' => function(Track $track) {
+                return sprintf('%s (by %s)', $track->getSongTitle(), $track->getArtistName());
+            },
 
             'query_builder' => function(TrackRepository $trackRepository) {
                 return $trackRepository->createQueryBuilder('track');
